pkgs/games/kobodeluxe/default.nix

```diff
@@ -9,11 +9,22 @@ stdenv.mkDerivation rec {
   };
 
   buildInputs = [ SDL SDL_image libGLU libGL ];
+  nativeBuildInputs = [ makeWrapper ];
 
   prePatch = ''
     sed -e 's/char \*tok/const char \*tok/' -i graphics/window.cpp
   '';
 
+  preConfigure = ''
+    substituteInPlace ./configure --replace "CONFIGDIR='HOME>>'" "CONFIGDIR='HOME>>/.config/${pname}/'";
+  '';
+
+  postInstall = ''
+    wrapProgram $out/bin/${meta.mainProgram} \
+      --run "mkdir -p ~/.config/${pname}" \
+      --add-flags "-scores ~/.config/${pname}"
+  '';
```

Comment:
A comment why this is necessary would be nice.

Reply:
Do you mean adding a comment in the source? Sure.
Do you think saving the scores in ~/.config/kobodeluxe is still ok?

Reply:
Is that *necessary* for operation (i.e. it fixes some issue)?

Reply:
Yes, it fixes https://github.com/NixOS/nixpkgs/issues/29111